scripts/EGAP/create_EGAP_json.py

```diff
@@ -27,7 +27,6 @@
     {'q6': 'B4 FACULTY MEMBER?'},
     {'q8': 'B5 PROSPECTIVE OR RETROSPECTIVE?'},
     {'q10': 'B6 EXPERIMENTAL STUDY?'},
-    {'q11': 'B7 DATE OF START OF STUDY'},
     {'q12': 'B8 GATE DATE'},
```

Comment:
It looks like Q12 was removed as well from the new version. Do we need to remove this line?

Reply:
I need the gate date for the embargo, but I pop it on this line https://github.com/corbinSanders/osf.io/blob/484bc5d9c139588207dcbd0173071866391fded9/osf/management/commands/import_EGAP.py#L262 from the metadata.